src/containers/Tenant/Query/ExecuteResult/ExecuteResult.tsx

```diff
@@ -274,6 +274,7 @@ export function ExecuteResult({
                             </Button>
                         </React.Fragment>
                     ) : null}
+                    <TraceButton traceId={data?.traceId} />
```

Comment:
Please, add `ClusterModeGuard` here (or inside `TraceButton` component). There is no meta for embedded version, there is no need to trigger clusterInfo query at all

Reply:
added
but for test set it to single